components/work_templates/index.tsx

```diff
@@ -157,6 +157,7 @@ const WorkTemplateModal = () => {
 
     const handleOnNameChanged = (name: string) => {
         trackEvent(TELEMETRY_CATEGORIES.WORK_TEMPLATES, 'customize_name', {category: selectedTemplate?.category, template: selectedTemplate?.id});
+        setErrorText('');
```

Comment:
nit: For consistency on the effect being used to reset error, could instead add `selectedName` as dependency:
https://github.com/mattermost/mattermost-webapp/blob/4635e9152ecc5ff46fca55bc5933494c77778299/components/work_templates/index.tsx#L119-L122

Reply:
+1 with @julmondragon but not nit from me :P

Reply:
thanks for the suggestion
done